app/verify/success.js

```diff
@@ -0,0 +1,16 @@
+const storage = require('../storage')
+
+const success = async (pendingFilenames, processedFilenames) => {
+  console.log('Renaming files')
+  await storage.renameFile(pendingFilenames.controlFilename, processedFilenames.controlFilename)
+  await storage.renameFile(pendingFilenames.batchFilename, processedFilenames.batchFilename)
+  await storage.renameFile(pendingFilenames.checksumControlFilename, processedFilenames.checksumControlFilename)
+  await storage.renameFile(pendingFilenames.checksumFilename, processedFilenames.checksumFilename)
+  console.log('Archiving files')
+  await storage.archiveFile(processedFilenames.checksumControlFilename)
```

Comment:
We're not archiving `pendingFilenames.batchFilename`

Reply:
Yes that's right.  The batch file with the payments in must remain in `batch/inbound`.  It's then picked up by `ffc-pay-batch-processor`.  

The other three aren't needed once verification is complete, so we can just archive those.